support-frontend/assets/helpers/redux/commonState/selectors.ts

```diff
@@ -55,3 +55,16 @@ export function isUserInAbVariant(abTestName: string, variantName: string) {
 		return participations[abTestName] === variantName;
 	};
 }
+
+export function isUserInAnyAbVariant(
```

Comment:
Is this function `isUserInAnyAbVariant` used anywhere?

Reply:
No, was umming/ahhing whether to keep this. Thought was useful to check 2+ variants at the same time. Can remove however as it is clutter. Presume if not used, just remove is the rule?

Reply:
Yeah i'd generally remove unused code to keep the codebase tidy, if we did keep it I'd suggest we add a comment, or even better unit tests to demonstrate it's purpose. If we're unsure I'd err on the side of removing unused code, and then revisit the decision if the need for the export arises in the future.